### PR TITLE
Coming soon componenet and "join us" page finish. small problem left

### PR DESCRIPTION
**Pull request by qimingliu1021**
on mobile frame version. some blank left on right side. Only on computer responsive design development, when screen < 640px (tailwind css sm) smallest screen, there's some blank left.

### CODE REVIEW

File: numaira_landing/src/app/(pages)/contact/page.tsx [hunk 1]
```diff
@@ -1,10 +1,410 @@
-'use client'
-import React from 'react'
+"use client";
+import React from "react";
+import { useState } from "react";
+import { BiChevronUp, BiChevronDown, BiCheck } from "react-icons/bi";
```
Comment (Yunzez) [inline numaira_landing/src/app/(pages)/contact/page.tsx:4]: don't use third party icons, use our own icon

File: numaira_landing/src/app/(pages)/contact/page.tsx [hunk 2]
```diff
@@ -1,10 +1,410 @@
...
+  const handleMouseLeave = (key, index) => {
+    setTimeZone((prev) => ({
+      ...prev,
+      [key]: prev[key].map((value, i) => (i === index ? false : value)),
+    }));
+  };
+
+  const setItemSelected = (key, index) => {
+    console.log(`In setItemSelected. Setting ${key} to true...`);
+    console.log(`current timeZone is: ${timeZone["EMEA"]}`);
```
Comment (Yunzez) [inline numaira_landing/src/app/(pages)/contact/page.tsx:47]: delete console log

File: numaira_landing/src/app/(pages)/contact/page.tsx [hunk 3]
```diff
@@ -1,10 +1,410 @@
...
+    setTimeZone((prev) => {
+      const updatedTimeZone = Object.keys(prev).reduce((acc, item) => {
```
Comment (Yunzez) [inline numaira_landing/src/app/(pages)/contact/page.tsx:49]: do not use reduce in simple page

File: numaira_landing/src/app/components/comming-soon.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+import formats from "../formats";
+
+const ComingSoon = () => {
+  return (
+    <div>
+      <div className="flex flex-col justify-center items-center h-96">
+        <LogoSVG width={80} height={80} fill={Colors.brand500} />
+        <div className="flex flex-col gap-2">
+          <div
+            className="font-semibold"
```
Comment (Yunzez) [inline numaira_landing/src/app/components/comming-soon.tsx:13]: repeat with the font weight in style

File: .vscode/settings.json [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
```
Comment (Yunzez) [inline .vscode/settings.json:2]: what is this?

Comment (qimingliu1021) [reply .vscode/settings.json:2]: deleted the file. worked fine

File: numaira_landing/src/app/(pages)/contact/page.tsx (hunk 2 above)
Comment (qimingliu1021) [reply numaira_landing/src/app/(pages)/contact/page.tsx:47]: changed using own icon

File: numaira_landing/requirements.txt [hunk 6]
```diff
@@ -0,0 +1 @@
+@ant-design/icons@4.0.0
```
[the remark this answers is not in the document]
Comment (qimingliu1021) [reply numaira_landing/requirements.txt:1]: changed with own icon

File: numaira_landing/src/app/components/comming-soon.tsx (hunk 4 above)
Comment (qimingliu1021) [reply numaira_landing/src/app/components/comming-soon.tsx:13]: deleted and keep what is in style

**Pull request merged**